utils/src/server/rpc_module.rs

```diff
@@ -374,13 +369,56 @@ impl Methods {
 		rx.next().await
 	}
 
+	/// Perform a method call and receive further subscriptions.
+	///
+	/// Returns a response to actual method call and a stream to process
+	/// futher subscriptions if a subcription was registered on the call.
+	///
```

Comment:
Worth mentioning what the intended use case for this is? "In-memory RPC queries, e.g. in tests"?